backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/OktaRepository.java

```diff
@@ -17,6 +17,16 @@
  * <p>Handles all user/organization management in Okta
  */
 public interface OktaRepository {
+  Integer OKTA_ORGS_LIMIT = 200;
+  Integer OKTA_RATE_LIMIT_SLEEP_MS = 60000;
+
+  default Integer getOktaOrgsLimit() {
```

Comment:
⚠️ defined methods to return the value so I can mock these values in the test -- let me know if there are other ideas for this 😅